[2022-04-06 20:52:19,565] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: divvybikes_elt_pipeline.bq_staging_table_task 2022-02-01T00:00:00+00:00 [None]>
[2022-04-06 20:52:19,582] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: divvybikes_elt_pipeline.bq_staging_table_task 2022-02-01T00:00:00+00:00 [None]>
[2022-04-06 20:52:19,582] {taskinstance.py:1094} INFO - 
--------------------------------------------------------------------------------
[2022-04-06 20:52:19,582] {taskinstance.py:1095} INFO - Starting attempt 2 of 3
[2022-04-06 20:52:19,582] {taskinstance.py:1096} INFO - 
--------------------------------------------------------------------------------
[2022-04-06 20:52:19,594] {taskinstance.py:1114} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bq_staging_table_task> on 2022-02-01T00:00:00+00:00
[2022-04-06 20:52:19,599] {standard_task_runner.py:52} INFO - Started process 13118 to run task
[2022-04-06 20:52:19,603] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'divvybikes_elt_pipeline', 'bq_staging_table_task', '2022-02-01T00:00:00+00:00', '--job-id', '244', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/extract_load.py', '--cfg-path', '/tmp/tmp1mzk72h2', '--error-file', '/tmp/tmpxpt5do0e']
[2022-04-06 20:52:19,605] {standard_task_runner.py:77} INFO - Job 244: Subtask bq_staging_table_task
[2022-04-06 20:52:19,634] {logging_mixin.py:109} INFO - Running <TaskInstance: divvybikes_elt_pipeline.bq_staging_table_task 2022-02-01T00:00:00+00:00 [running]> on host c653229d7f92
[2022-04-06 20:52:19,670] {taskinstance.py:1253} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=divvybikes_elt_pipeline
AIRFLOW_CTX_TASK_ID=bq_staging_table_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=backfill__2022-02-01T00:00:00+00:00
[2022-04-06 20:52:19,672] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-06 20:52:19,683] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.6/site-packages/***/providers/google/cloud/hooks/bigquery.py:637 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.create_empty_table` method withpass passing the `table_resource` object. This gives more flexibility than this method.
[2022-04-06 20:52:19,683] {taskinstance.py:1462} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1164, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1282, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1312, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 1215, in execute
    encryption_configuration=self.encryption_configuration,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 689, in create_external_table
    table = Table(table_ref=TableReference.from_string(external_project_dataset_table, project_id))
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/table.py", line 182, in from_string
    table_id, default_project=default_project, property_name="table_id"
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/_helpers.py", line 766, in _parse_3_part_id
    "got {}".format(full_id, property_name=property_name)
ValueError: table_id must be a fully-qualified ID in standard SQL format, e.g., "project.dataset.table_id", got divvybikes_staging_table
[2022-04-06 20:52:19,685] {taskinstance.py:1512} INFO - Marking task as UP_FOR_RETRY. dag_id=divvybikes_elt_pipeline, task_id=bq_staging_table_task, execution_date=20220201T000000, start_date=20220406T205219, end_date=20220406T205219
[2022-04-06 20:52:19,736] {local_task_job.py:151} INFO - Task exited with return code 1
[2022-04-06 20:52:19,752] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
